### PR TITLE
#2211 - Vertices list of minkowski sum of zonotopes

### LINKED ISSUE #2211: Vertices list of minkowski sum of zonotopes
The following is not implemented:

```
X = rand(Zonotope)
Y = rand(Zonotope)
vertices_list(X + Y)
```

### PR DESCRIPTION
**Pull request by SebastianGuadalupe**
Closes #2211.

### CODE REVIEW

File: src/Interfaces/AbstractZonotope.jl [hunk 1]
```diff
@@ -546,3 +546,22 @@ function constraints_list(Z::AbstractZonotope{N}; check_full_rank::Bool=true
...
+
+Return the list of constraints defining a zonotopic set.
+
+### Input
+
+- `MS` -- minkowski sum of two zonotopic sets
+
+### Output
+
+The list of vertices of the minkowski sum of two zonotopic set.
```
Comment (schillic) [inline src/Interfaces/AbstractZonotope.jl:562]:
```suggestion
Return the list of vertices for the Minkowski sum of two zonotopic sets.

### Input

- `MS` -- Minkowski sum of two zonotopic sets

### Output

The list of vertices of the Minkowski sum of two zonotopic sets.
```

File: test/unit_Zonotope.jl [hunk 2]
```diff
@@ -242,6 +242,11 @@ for N in [Float64]
     Z = convert(Zonotope, S)
     @test vertices_list(Z) == [element(S)]
 
+    #vertices for minkowski sum of zonotopic sets
+    X = Zonotope([0, 0.], [[1, 0.]])
+    Y = Zonotope([0, 0.], [[1, 1.]])
+    @test vertices_list(X + Y) == [[2.0, 1.0], [0.0, 1.0], [-2.0, -1.0], [0.0, -1.0]]
```
Comment (schillic) [inline test/unit_Zonotope.jl:248]:
```suggestion
    X = Zonotope(N[0 0], [N[1, 0]])
    Y = Zonotope(N[0, 0], [N[1, 1]])
    @test ispermutation(vertices_list(X + Y), [N[2, 1], N[0, 1], N[-2, -1], N[0, -1]])
```

File: test/unit_Zonotope.jl [hunk 3]
```diff
@@ -242,6 +242,11 @@ for N in [Float64]
     Z = convert(Zonotope, S)
     @test vertices_list(Z) == [element(S)]
 
+    #vertices for minkowski sum of zonotopic sets
+    X = Zonotope(N[0 0], [N[1, 0]])
```
Comment (schillic) [inline test/unit_Zonotope.jl:246]:
```suggestion
    X = Zonotope(N[0, 0], [N[1, 0]])
```
oops

File: docs/src/lib/lazy_operations/MinkowskiSum.md [hunk 4]
```diff
@@ -18,6 +18,7 @@ isbounded(::MinkowskiSum)
 isempty(::MinkowskiSum)
 constraints_list(::MinkowskiSum)
 ∈(::AbstractVector{N}, ::MinkowskiSum{N, S1, S2}) where {N, S1<:AbstractSingleton, S2<:LazySet}
+vertices_list(MS::MinkowskiSum{N, <:AbstractZonotope{N}, <:AbstractZonotope{N}}) where{N<:Real}
```
Comment (mforets) [inline docs/src/lib/lazy_operations/MinkowskiSum.md:21]:
```suggestion
vertices_list(MS::MinkowskiSum{N, Z1, Z2}) where{N<:Real, Z1<:AbstractZonotope{N}, Z2<:AbstractZonotope{N}}
```

File: src/LazyOperations/MinkowskiSum.jl [hunk 5]
```diff
@@ -252,3 +252,20 @@ end
...
+
+### Input
+
+- `ms` -- Minkowski sum of two zonotopic sets
+
+### Output
+
+The list of vertices of the Minkowski sum of two zonotopic sets.
+"""
+function vertices_list(v::MinkowskiSum{N, Z1, Z2}) where {N<:Real, Z1<:AbstractZonotope{N}, Z2<:AbstractZonotope{N}}
```
Comment (mforets) [inline src/LazyOperations/MinkowskiSum.jl:269]:
```suggestion
function vertices_list(ms::MinkowskiSum{N, Z1, Z2}) where {N<:Real, Z1<:AbstractZonotope{N}, Z2<:AbstractZonotope{N}}
```

**Pull request merged**